client/my-sites/checkout/concierge-quickstart-session/index.jsx

```diff
@@ -62,6 +63,14 @@ export class ConciergeQuickstartSession extends React.Component {
 			comment: '"Checkout" is the part of the site where a user is preparing to make a purchase.',
 		} );
 
+		// If the param value is 'add', then we will add the concierge session item to cart and proceed to checkout,
+		// without showing any UI. We use this method when a logged out user clicks 'Get Started' on the concierge
+		// offer page.
+		if ( 'add' === receiptId ) {
```

Comment:
`receiptId ` is an unfortunate name but I am following up with another PR that cleans up this function and will do away with the need for a receiptId, so that should take care of this.

Reply:
👍 Just noting that the receiptId was only ever needed to be passed through to the thank you page where it was used to display some plan specific information.